src/components/asset-list/RecyclerAssetList2/FastComponents/FastBalanceCoinRow.tsx

```diff
@@ -0,0 +1,323 @@
+      : theme.colors.blueGreyDark50;
+
+    const nativeDisplay = item?.balance?.display;
+
+    const valueColor = nativeDisplay
+      ? theme.colors.dark
+      : theme.colors.blueGreyLight;
+
+    return (
+      <View style={cx.flex}>
+        <ButtonPressAnimation
+          onPress={handlePress}
+          scaleTo={0.96}
+          testID={`balance-coin-row-${item.name}`}
+        >
+          <View style={[cx.container]}>
+            <FastCoinIcon
+              address={item.mainnet_address ?? item.address}
+              assetType={item.type}
+              symbol={item.symbol}
+              theme={theme}
+            />
+
+            <View style={[cx.innerContainer, isHidden && cx.hiddenRow]}>
+              <View style={cx.row}>
+                <View style={cx.textWrapper}>
+                  <Text
+                    align="left"
+                    numberOfLines={1}
+                    size="16px"
+                    weight="medium"
+                  >
+                    {item.name}
+                  </Text>
+                </View>
+
+                <Text
+                  align="left"
+                  color={{ custom: valueColor }}
+                  size="16px"
+                  weight="medium"
+                >
+                  {item?.native?.balance?.display ??
+                    `${nativeCurrencySymbol}0.00`}
+                </Text>
+              </View>
+
+              <View style={[cx.row, cx.bottom]}>
+                <View style={cx.textWrapper}>
+                  <Text
+                    align="left"
+                    color={{ custom: theme.colors.blueGreyDark50 }}
+                    numberOfLines={1}
+                    size="14px"
+                  >
+                    {nativeDisplay ?? ''}
+                  </Text>
+                </View>
+
+                <Text color={{ custom: changeColor }} size="14px">
+                  {percentageChangeDisplay}
+                </Text>
+              </View>
+            </View>
+          </View>
+        </ButtonPressAnimation>
+      </View>
+    );
+  }
+);
+
+MemoizedBalanceCoinRow.displayName = 'MemoizedBalanceCoinRow';
+
+export default React.memo(function BalanceCoinRow({
+  uniqueId,
+  extendedState,
+}: {
+  uniqueId: string;
+  extendedState: ExtendedState;
+}) {
+  const {
+    theme,
+    nativeCurrencySymbol,
+    navigate,
+    nativeCurrency,
+    hiddenCoins,
+    pinnedCoins,
+    toggleSelectedCoin,
+    isCoinListEdited,
+  } = extendedState;
+
+  const onPress = useCallback(() => {
+    toggleSelectedCoin(uniqueId);
+  }, [uniqueId, toggleSelectedCoin]);
+
+  // HACK to make sure we don't rerender MemoizedBalanceCoinRow
+  // when isCoinListEdited === true and we need to change onPress callback
+  const maybeCallback = useRef<null | (() => void)>(null);
+  maybeCallback.current = isCoinListEdited ? onPress : null;
+
+  // we return false to make sure we don't run includes for each row when it's not needed
```

Comment:
what?

Reply:
it was because of `includes` now we don't have includes so removed it